### PR TITLE
Add StorageComparison Test

### PR DESCRIPTION
**Pull request by JVBB83**
Added a new test that compares a vehicle stored in memory and in a file to ensure all the information has been stored correctly in both methods.

### CODE REVIEW

File: BusinessCore.Tests/JVBBTests.cs [hunk 1]
```diff
@@ -9,6 +12,89 @@ namespace BusinessCore.Tests
...
     {
+        public string VehiclesFilePath
+        {
+            get
+            {
+                return Path.Combine(Environment.CurrentDirectory, "Vehicles.xml");
+            }
+        }
+
+        public bool SameVehicle(Vehicle v1, Vehicle v2)
```
Comment (hossmi) [inline BusinessCore.Tests/JVBBTests.cs]: Please, refactor this method using **fail faster** approach.
Make private, static and move it at the end of the class.

Comment (JVBB83) [reply BusinessCore.Tests/JVBBTests.cs]: Done, done, done and done :)

**Comment by hossmi** [comment]
hi @JVBB83 , you need to update this branch with last changes from **develop**.

**Comment by JVBB83** [comment]
Merged with current develop, test StorageComparison has been updated to comply with the changes.

**Pull request merged**